var/spack/.gitignore

```diff
@@ -0,0 +1,2 @@
+bootstrap/
+cache/
```

Comment:
Why is this second line needed, when we already have https://github.com/spack/spack/blob/5b1394f1cfd5196f0ffca31032ab5109a2836ec0/.gitignore#L6

Reply:
I see, will move it to the global ignore for consistency